### ISSUE TITLE
Multiple WotLK Dungeon Set 2 sets are not showing

### ISSUE THREAD

**Issue by Citano** [SLOKnightfall/BetterWardrobe#140]
Seems like multiple WotLK Dungeon Set 2 sets are not showing under the Extra tab in the Appearance section.
They are as follows: 
Brazen Trespass Armor - LEATHER
Snowy Bramble Armor - LEATHER
Ravenous Fiend Set - MAIL
Abandoned Fealty Battleplate - PLATE

Two sets in this category are showing as intended
Ravenous Fiend Set (Recolor) - MAIL
Abandoned Fealty Battleplate (Recolor) - PLATE

No cloth sets mentioned, since they don't have a WotLK Dungeon Set 2 set at all.

**Comment by SLOKnightfall** [comment]
Thanks for letting me know.  I checked and the set data is there, but a lot of the sets were flagged as class only due to WoWhead listing class specific legendaries as the prime set item for that slot.  The still should be showing up if you have the "Class set" filter disabled, but for some reason the sets are only showing for the classes they are flagged for.  I'll look into why they are not being show with the filter disabled and update the sets in my DB to remove the class flags.

**Comment by Citano** [comment]
Great to hear, however when I tried to toggle off "Class Sets Only", they still didn't show.

I also found some more sets from TBC that has the same issue, such as ["Dragonhide Battlegear"](https://www.wowhead.com/transmog-set=951/dragonhide-battlegear) and ["Opportunist's Battlegear"](https://www.wowhead.com/transmog-set=791/opportunists-battlegear).
Both are Leather and on Wowhead they are listed as "Level 70 PvP Rare Set 2"
Another example is the ["Mooncloth Battlegear"](https://www.wowhead.com/transmog-set=771/mooncloth-battlegear) Cloth set, another "Level 70 PvP Rare Set 2"

Guess this is the same issue?

![](https://i.gyazo.com/f6b9bc10f529356c179d213fae70c5d2.png)
